core/src/test/java/com/sequenceiq/cloudbreak/service/image/ImageCatalogServiceTest.java

```diff
@@ -877,6 +891,60 @@ public void testGetImageShouldLookupCustomImageInCaseOfNullImageCatalogUrl()
         assertEquals(statedImage.getImage(), actual.getImage());
     }
 
+    @Test
+    public void testFindAllByIdsWithDefaultsInCaseOfCustomCatalogsOnly() {
+        ImageCatalog customImageCatalog = new ImageCatalog();
```

Comment:
This first "block" is duplicated in the next test case. Please create a method like getImageCatalogs and you can use it in both test cases.